src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -70,6 +70,23 @@ static int _determinRecordCap(const AST *ast) {
     return recordsCap;
 }
 
+int CondTraverseToString(const OpBase *ctx, char *buff, uint buff_len) {
+    const CondTraverse *op = (const CondTraverse*)ctx;
+
+    int offset = 0;    
+    offset += snprintf(buff + offset, buff_len-offset, "%s | ", op->op.name);
+    offset += Node_ToString(op->algebraic_expression->src_node, buff + offset, buff_len - offset);
+    if(op->algebraic_expression->edge) {
+        offset += snprintf(buff + offset, buff_len-offset, "-");
+        offset += Edge_ToString(op->algebraic_expression->edge, buff + offset, buff_len - offset);
```

Comment:
Doesn't handle multiple edge types, see comment in `edge.h`.

Reply:
You mean [e:country|continent]?